integration_test/cases/repo.exs

```diff
@@ -80,16 +80,17 @@ defmodule Ecto.Integration.RepoTest do
   test "insert, update and delete" do
     post = %Post{title: "create and delete single", text: "fetch empty"}
 
-    assert %Post{} = TestRepo.insert(post)
+    assert %Post{__state__: :loaded} = TestRepo.insert(post)
```

Comment:
I would be more comfortable if we added those to `test/ecto/repo_test.exs` as they are not part of the integration test per se. Thoughts?